docs/api/modules/store.md

```diff
@@ -0,0 +1,182 @@
+
+The `StoreViewport` [type](https://github.com/dvisionlab/Larvitar/blob/master/imaging/types.d.ts#L34) includes the following key properties:
+
+| Property                      | Type          | Description                                                               |
+|-------------------------------|---------------|---------------------------------------------------------------------------|
+| `loading` 	                | `number`	    | Caching progress of the series from 0% to 100 %, initialized to null.     |
+| `ready`                       | `boolean`	    | True when the viewport is ready and imageId has been rendered.            |
+| `seriesUID`                   | `string?`	    | Unique identifier for the series.                                         |
+| `modality`                    | `string`	    | Modality of the image (e.g., "CT").                                       |
+| `isColor`                     | `boolean`	    | True if the image is in color.                                            |
+| `isMultiframe`                | `boolean`	    | True if the image is multiframe.                                          |
+| `isTimeserie`                 | `boolean`	    | True if the image is a timeserie.                                         |
+| `isDSAEnabled`                | `boolean`	    | True if DSA is enabled.                                                   |
+| `isPDF`                       | `boolean`	    | True if the image is a PDF.                                               |
+| `imageId`                     | `string`	    | Identifier for the image.                                                 |
+| `rows`                        | `number`	    | Rows in the image.                                                        |
+| `cols`                        | `number`	    | Columns in the image.                                                     |
+| `spacing_x`                   | `number`	    | Spacing in the x direction.                                               |
+| `spacing_y`                   | `number`	    | Spacing in the y direction.                                               |
+| `thickness`                   | `number`	    | Slice thickness.                                                          |
+| `numberOfSlices`              | `number?`	    | Number of slices in the image.                                            |
+| `numberOfFrames`              | `number?`	    | Number of frames in the image.                                            |
+| `minPixelValue`               | `number`	    | Minimum pixel value.                                                      |    
+| `maxPixelValue`               | `number`	    | Maximum pixel value.                                                      |
+| `sliceId`                     | `number`	    | Slice index identifier.                                                   |
```

Comment:
We should add a better (stricter) explanation of what "sliceId" is. The index of the image in the imageIds array ? Or the index of the image in the volume ?

Reply:
The index of the image in the imageIds array is equal to the index of the image in the volume since imageIds array is ordered from 0 to n-1. But yes, it is the index of the image to be displayed from a user-point-of-view